### PR TITLE
Added missing rel attribute in href

### LINKED ISSUE #7410: Links to cross-origin destinations are unsafe
```
<a class="nav-link" target="_blank" href="//store.publiclab.org" >
```

When you link to a page on another site using the `target="_blank"` attribute, you can expose your site to performance and security issues:

- The other page may run on the same process as your page. If the other page is running a lot of JavaScript, your page's performance may suffer.

- The other page can access your window object with the `window.opener` property. This may allow the other page to redirect your page to a malicious URL.

**Adding `rel="noopener"` or `rel="noreferrer"` to your `target="_blank"` links avoids these issues.**

> `noreferrer`: It is used to Specify that the browser should not send an HTTP referer header if the user follows the hyperlink.

### PR DESCRIPTION
**Pull request by vinitshahdeo**
Fixes #7410

Changes made: 
Added missing `rel="noreferrer"` attribute in `<a></a>` tag.

---
* [X] PR is descriptively titled 📑 and links the original issue above 🔗
* [X] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [X] code is in uniquely-named feature branch and has no merge conflicts 📁

Thanks!

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -76,6 +76,8 @@
           </div>
         </li>
 
+        <li class="nav-item d-sm-none d-lg-inline">
+          <a class="nav-link" target="_blank" href="//store.publiclab.org" rel="noreferrer"><%= t('layout._header.store') %></a>
         <li class="nav-item d-sm-none d-lg-inline" style="padding:0 10px;">
           <a class="nav-link btn btn-outline-secondary" target="_blank" href="//store.publiclab.org"><%= t('layout._header.store') %></a>
```
Comment (VladimirMikulic) [inline app/views/layouts/_header.html.erb]: Missing `noreferrer` on line 82.

Comment (Uzay-G) [reply app/views/layouts/_header.html.erb]: Nice catch!

Comment (vinitshahdeo) [reply app/views/layouts/_header.html.erb]: @VladimirMikulic Added missing `noreferrer`. Thanks:)

**Comment by vinitshahdeo** [comment]
@Uzay-G Thanks:)

**Comment by SidharthBansal** [comment]
@VladimirMikulic review pls

**Comment by VladimirMikulic** [comment]
@vinitshahdeo could you rebase the branch and push again? `git rebase master; git push`

**Comment by vinitshahdeo** [comment]
@VladimirMikulic Rebased with `master`. Please merge [this](https://github.com/publiclab/plots2/pull/7373) PR #7373  before merging it.

**Comment by VladimirMikulic** [comment]
@vinitshahdeo are you sure that you've rebased this? You've introduced 300+ commits?

**Pull request closed without merging**